lib/bodyScrollLock.es6.js

```diff
@@ -99,14 +99,16 @@ const setPositionFixed = () => window.requestAnimationFrame(() => {
     previousBodyPosition = {
       position: document.body.style.position,
       top: document.body.style.top,
-      left: document.body.style.left
+      left: document.body.style.left,
+      right: document.body.style.right
     };
 
     // Update the dom inside an animation frame
     const { scrollY, scrollX, innerHeight } = window;
     document.body.style.position = 'fixed';
     document.body.style.top = `${-scrollY}px`;
     document.body.style.left = `${-scrollX}px`;
+    document.body.style.right = '0px';
```

Comment:
Is not needed set measure unit when value equals of 0

Reply:
Good point! Fixed!